src/latin_language_model.js

```diff
@@ -146,5 +146,32 @@ class LatinLanguageModel extends LanguageModel {
   static toCode () {
     return Constants.STR_LANG_CODE_LAT
   }
+
+  /**
+   * Sets inflection grammar properties based on its characteristics
+   * @param {Inflection} inflection - An inflection object
+   * @return {Object} Inflection properties
+   */
+  static getInflectionGrammar (inflection) {
+    let grammar = {
+      fullFormBased: false,
+      suffixBased: false,
+      pronounClassRequired: false
+    }
+    if (inflection.hasOwnProperty(Feature.types.part) &&
+      Array.isArray(inflection[Feature.types.part]) &&
+      inflection[Feature.types.part].length === 1) {
+      let partOfSpeech = inflection[Feature.types.part][0]
+      if (partOfSpeech.value === Constants.POFS_PRONOUN) {
```

Comment:
I'm wondering if we can combine both approaches here. If we know for sure by part of speech (pronoun), or by some other grammar property that a word is going to be irregular, we can mark it this way immediately. However, if we can't say for sure (i.e. verbs that can be both regular and irregular), then probably we can do a lookup within forms data and decide based on that.

In order to keep it within a language model, we can create a function there. It could be something like that (in pseudocode):
```
function isIrregular(inflection, formsData) {
  if (inflection.partOfSpeech === 'pronoun') { return true }
  else { return lookupFormsData(word) }
}
```

What do you think?

Reply:
yes that should work. I don't really know what is best here, so having the flexibility to handle it both ways sounds good.